package.json

```diff
@@ -26,8 +26,11 @@
     "clean": "gts clean",
     "compile": "tsc",
     "fix": "gts fix",
-    "test": "npm run test-unit",
-    "test-ts": "node --experimental-top-level-await --experimental-loader ./src/loader.mjs ../presm/tests/tests.ts",
-    "test-unit": "node --experimental-top-level-await ./tests/tests.mjs"
+    "test": "npm run test-unit && npm run test-system",
+    "test-snapshot": "npm run test-snapshot-unit && npm run test-snapshot-system",
+    "test-unit": "tap --no-coverage ./test/unit/",
```

Comment:
I'm curious about why the `--no-coverage` - is it just that there aren't enough tests yet, and we want to get TAP working first?

Reply:
Allowing coverage triggers TAP's use of [`spawn-wrap`](https://www.npmjs.com/package/spawn-wrap).  This doesn't break the loader but adds more for the hooks to deal with imports so the snapshot gets messy.  Added `--no-coverage` to keep this test simple for now.  Good question though, I can explore coverage testing later.